ecadlabs/taqueria
# ✨ Feature ➾ Provide LIGO syntax highlighting support in VSCode extension

# 🌮 Taqueria PR

## 🪁 Description

This PR adds built-in support for LIGO and Michelson languages syntax highlighting.
You don't need to install LIGO extension or some plugin for Michelson - all comes working in Taqueria VsCE 👀 

## 🪂 Pre-Merge Checklist (Definition of Done)

🚦 Required to merge:

- [x] ⛱️ I have completed this PR template in full and updated the title appropriately
- [x] ⛵ My code builds cleanly, and I have manually tested the changes
- [ ] 🏄‍♂️ Another team member has built this branch and done manual testing on the change
- [ ] 🏖️ New and existing unit tests pass locally and in CI
- [ ] 🔱 The test plan has been implemented and verified by an SDET
- [ ] 🦀 Automated tests have been written and added to this PR
- [x] 🤿 Corresponding changes have been made to all documentation
- [x] 🐚 Required changes to the VScE have been made
- [x] 🪸 Required updates to scaffolds have been made
- [ ] 🚢 The release checklist has been completed

## 🛩️ Summary of Changes

- Added entries in package.json to register new languages support by the extension
- Incorporated TextMate language definitions for LIGO dialects and Michelson

## 🎢 Test Plan

1. Open taqueria-enabled repo with TVsCE enabled
2. Open LIGO file
3. Open Michelson file
4. Check if the syntax is properly highlighted 👀 

## 🛸 Type of Change

- [ ] 🧪 Pre-Release ➾
